py3status/modules/kdeconnector.py

```diff
@@ -200,6 +213,30 @@ def _get_battery(self):
 
         return battery
 
+    def _get_conn(self):
+        """
+        Get the connection report
+        """
+        try:
+            if self._con:
+                # Possible values are 0 - 4
+                strength = self._con.cellularNetworkStrength
+                type = self._con.cellularNetworkType
+
+                con_info = {
+                    "strength": f"{int((strength / 4) * 100)}%",
```

Comment:
Ideally, this should stay as an int and to drop `%` and having users add `%` themselves if they want it.

Reply:
Number 0 - 4 will tell nothing until you know that max value is 4. That is why I finally decided to calculate it to %.

Reply:
Not what I was trying to suggest. I want to remove `%`. 

`{strength}%` is better than adding `%` to end of `strength` value.

As an example, look at  `charge` placeholder where the value remains a number and the users can customize if they want `%` or not in the `format`.

```python
format = "{name}{notif_status} {bat_status} {charge}%" # default format
format = "{name}{notif_status} {bat_status} {charge}% {strength}%"  # percent symbols
format = "{name}{notif_status} {bat_status} {charge} {strength}"  # no percent symbols
```